### PR TITLE
gauche: Add version 0.9.9

### PR DESCRIPTION
**Pull request by qykth-git**
Add Gauche manifest.
See also: https://github.com/shirok/Gauche/issues/539

### CODE REVIEW

File: bucket/gauche.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "##": "This file is generated from Gauche source tree",
```
Comment (Ash258) [inline bucket/gauche.json]:
```suggestion
```

File: bucket/gauche.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "description": "Scheme Scripting Engine",
+    "homepage": "https://practical-scheme.net/gauche/",
+    "version": "0.9.9",
+    "license": {
+        "identifier": "BSD-3-Clause",
+        "url": "https://github.com/shirok/Gauche/blob/master/COPYING"
+    },
```
Comment (Ash258) [inline bucket/gauche.json]:
```suggestion
    "license": "BSD-3-Clause",
```

File: bucket/gauche.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        "64bit": {
+            "url": "https://github.com/shirok/Gauche/releases/download/release0_9_9/Gauche-mingw-0.9.9-64bit.msi",
+            "hash": "620389588eda0765973bc499c7eb340744b4d463022cd4b3a37164539952eafc"
+        },
+        "32bit": {
+            "url": "https://github.com/shirok/Gauche/releases/download/release0_9_9/Gauche-mingw-0.9.9-32bit.msi",
+            "hash": "6b6f742f05c8153448f779850333eac9a24894e2a7c706aa6bfd230bc1435434"
+        }
+    },
+    "bin": [
```
Comment (Ash258) [inline bucket/gauche.json]:
```suggestion
    "extract_dir": "Gauche",
    "bin": [
```

File: bucket/gauche.json [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+        "Gauche/bin/gauche-cesconv.exe",
+        "Gauche/bin/gauche-config.exe",
+        "Gauche/bin/gauche-install.exe",
+        "Gauche/bin/gauche-package.exe",
+        "Gauche/bin/gosh.exe",
+        "Gauche/bin/gosh-noconsole.exe"
```
Comment (Ash258) [inline bucket/gauche.json]:
```suggestion
        "bin\\gauche-cesconv.exe",
        "bin\\gauche-config.exe",
        "bin\\gauche-install.exe",
        "bin\\gauche-package.exe",
        "bin\\gosh.exe",
        "bin\\gosh-noconsole.exe"
```

File: bucket/gauche.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    ],
+    "checkver": {
+        "github": "https://github.com/shirok/Gauche"
```
Comment (Ash258) [inline bucket/gauche.json]:
```suggestion
        "github": "https://github.com/shirok/Gauche",
        "regex": "mingw-([\\d.]+)-\\d+bit\\.msi"
```

File: bucket/gauche.json [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        }
+    },
+    "bin": [
+        "Gauche/bin/gauche-cesconv.exe",
+        "Gauche/bin/gauche-config.exe",
+        "Gauche/bin/gauche-install.exe",
+        "Gauche/bin/gauche-package.exe",
+        "Gauche/bin/gosh.exe",
+        "Gauche/bin/gosh-noconsole.exe"
+    ],
```
Comment (Ash258) [inline bucket/gauche.json]:
```suggestion
    ],
    "shortcuts": [
        [
            "bin\\gosh.exe",
            "Gosh"
        ]
    ],
```

**Comment by qykth-git** [comment]
@Ash258 
Update Gauche manifest as  your suggest.

**Pull request merged**